test/formatting.sh

```diff
@@ -6,6 +6,8 @@ ROOT=$(dirname "${BASH_SOURCE}")/..
 PACKAGES="${ROOT}/pkg/ ${ROOT}/cmd/"
```

Comment:
I don't like this in `test` folder... I'm not sure why it is here.   need a utils folder or `tools` folder.

Reply:
or the `hack` dir that is used for similar things

Reply:
this was here prior, I'll move it over.